### DISCUSSION TITLE
New GH Organization

### DISCUSSION THREAD

**Discussion by JoFrhwld** [JoFrhwld/FAVE#86] [General]
Just an FYI, I've created a new organization to which I'd like to move ownership of the primary FAVE repo: https://github.com/Forced-Alignment-and-Vowel-Extraction

I'm not exactly sure how smoothly things like moving over things like open PRs or keeping track of forks will go, so I think before making that move, I'll want to

- [x] review and merge #83 
- [ ] review and merge #85
- [ ] Fix up whatever's going on with the docs branches
    - [ ] Create a new `feature/doc-fix` branch
    - [ ] Cherrypick as appropriate from `feature/docs` and `feature/newdocs`

**Comment by chrisbrickhouse** [comment]
The branching issue has been hard to resolve so I support any solution short of a force-push. Long term, I think we want to cleanup stale branches, ideally down to `main`, `dev`, and `docs`. Then have a commit that they can all sync up on (probably a 2.x) and then we can start diverging again.

**Comment by chrisbrickhouse** [comment]
Also, as part of the migration, I'd suggest we migrate the default branch name to `main` instead of `master`.
